src/ui-config/networksConfig.ts

```diff
@@ -232,7 +239,12 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
   },
   [ChainId.harmony]: {
     name: 'Harmony',
-    publicJsonRPCUrl: ['https://api.s0.t.hmny.io', 'https://api.harmony.one'],
+    publicJsonRPCUrl: [
+      'https://harmony-0.gateway.pokt.network/62b3314e123e6f00397f19ca',
+      'https://api.s0.t.hmny.io',
+      'https://api.harmony.one',
+      ''
```

Comment:
```suggestion
```

Reply:
Removed this in [9](https://github.com/aave/interface/pull/892/commits/b465bc9cdf88a540da9c7fec3bd939591b3c1878)